sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -180,9 +180,18 @@ def _buildprovider(
     # pylint:disable=protected-access
     provider = AzureAppConfigurationProvider()
     headers = kwargs.pop("headers", {})
-    headers["Correlation-Context"] = _get_correlation_context(key_vault_options)
+
+    if os.environ.get(RequestTracingDisabledEnvironmentVariable, default="").lower() != "true":
+        headers["Correlation-Context"] = _get_correlation_context(key_vault_options)
+
     useragent = USER_AGENT
 
+    if "retry_total" not in kwargs:
+        kwargs["retry_total"] = 2
+
+    if "retry_backoff_max" not in kwargs:
+        kwargs["retry_backoff_max"] = 1
```

Comment:
retry_backoff_max is the max (seconds) between two retrys. Defaults to 120s. Do you really want to change it to 1?

Reply:
Oh... No should be 1 min. Will updated to 60s.